### PR TITLE
ATS-285 | IV uplift redirect

### PR DESCRIPTION
**Pull request by Synnnapse**

### CODE REVIEW

**Comment by Synnnapse** [comment]
Hey @oc-ste this is ready to review now whenever you're free 👍

File: app/controllers/auth/PayeAuthAction.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ class PayeAuthActionImpl @Inject()(override val authConnector: AuthConnector,
     implicit val hc: HeaderCarrier =
       HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))
 
-    authorised(ConfidenceLevel.L200 and AuthNino(hasNino = true))
+    authorised(ConfidenceLevel.L200 and AuthNino(hasNino = true) and CredentialStrength(CredentialStrength.strong))
```
Comment (oc-ste) [inline app/controllers/auth/PayeAuthAction.scala]: Need to do a MFA uplift if this fails and tests around it

You will also need a test for both, just so you know Individuals should go through MFA then IV.

File: conf/application.conf [hunk 2]
```diff
@@ -119,6 +119,11 @@ microservice {
       port = 9999
     }
 
+    identity-verification-frontend {
+      host = localhost
+      port = 9948
```
Comment (oc-ste) [inline conf/application.conf]: This should be to 9938? 9948 is the stubs port, which IVF will redirect to 9938

File: conf/application.conf [hunk 3]
```diff
@@ -145,8 +150,10 @@ microservice {
     yta.url = "http://localhost:9020/account"
     feedback.url = "http://localhost:9514/feedback/ATS/personal"
     login.url = "http://localhost:9025/gg/sign-in?continue=http://localhost:9217/annual-tax-summary"
-    login.paye.url = "http://localhost:9025/gg/sign-in?continue=http://localhost:9217/paye/annual-tax-summary"
     login-callback.url = "http://localhost:9217/annual-tax-summary"
+    paye.login.url = "http://localhost:9025/gg/sign-in"
```
Comment (oc-ste) [inline conf/application.conf]: Do you need a continue bit?

File: app/controllers/auth/PayeAuthAction.scala (hunk 1 above)
Comment (Synnnapse) [reply app/controllers/auth/PayeAuthAction.scala]: As discussed, in the case for credential strength other than strong, we'll just be catching the exception and logging that 👍

File: conf/application.conf (hunk 2 above)
Comment (Synnnapse) [reply conf/application.conf]: As discussed this points to the IV stub frontend 👍

File: conf/application.conf (hunk 3 above)
Comment (Synnnapse) [reply conf/application.conf]: The continue url was appended twice so I've just removed it from this url 👍

**Pull request merged**